src/test/scala/io/iohk/ethereum/crypto/ECDSASignatureSpec.scala

```diff
@@ -3,23 +3,39 @@ package io.iohk.ethereum.crypto
 import java.math.BigInteger
 
 import akka.util.ByteString
+import org.scalacheck.Arbitrary
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalatest.prop.PropertyChecks
 import org.scalatest.{FlatSpec, Matchers}
+import org.spongycastle.crypto.params.ECPublicKeyParameters
 import org.spongycastle.util.encoders.Hex
 
-class ECDSASignatureSpec extends FlatSpec with Matchers {
-  "ECDSASignature" should "recoverPubBytes correctly fro go ethereum transaction" in {
+class ECDSASignatureSpec extends FlatSpec with Matchers with PropertyChecks {
+  "ECDSASignature" should "recoverPubBytes correctly for go ethereum transaction" in {
```

Comment:
Update the description as `recoverPubBytes` is a private function and not used here.